src/validators/cache/index.js

```diff
@@ -1,6 +1,14 @@
-import {noop} from 'lodash'
+import {friendlyTypeOf} from '../../utils'
+
 export default {
   key: 'cache',
-  validate: noop,
+  validate: validateCache,
+}
+
+function validateCache(value) {
+  const type = friendlyTypeOf(value)
+  if (type !== 'boolean' && type !== 'object') {
+    return `Unexpected type ${type}. Must be a boolean or an object.`
```

Comment:
So the style guide for messages I formulated says no period at the end. Reason: when you have multiple errors joined by " - ", periods look awkward in front of the dash. I'm torn though, as I like it better with periods, apart from that corner case. What do you think?